bom-weekly/pom.xml

```diff
@@ -138,6 +138,11 @@
                 <artifactId>gitlab-branch-source</artifactId>
                 <version>640.v7101b_1c0def9</version>
             </dependency>
+            <dependency>
+                <groupId>io.jenkins.plugins</groupId>
+                <artifactId>ionicons-api</artifactId>
```

Comment:
It might be necessary to just drop the 2.332.x and older lines. I do not think `bom` as currently set up can handle managed plugins incompatible with older lines. This comes up for newly detached plugins too and we do not have a good solution.